frontend/src/pages/DraftIndex.tsx

```
@@ -0,0 +1,44 @@
+import { Container, Pagination } from '@mui/material'
+import { ChangeEvent } from 'react'
+import { useNavigate } from 'react-router'
+import { useSearchParams } from 'react-router-dom'
+import useSWR from 'swr'
+import { Error } from '../components/utilities/Error'
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)